emojivoto-voting-patch/api.go

```diff
@@ -0,0 +1,448 @@
+package api
+
+import (
+	"context"
+
+	pb "github.com/edgelesssys/emojivoto/emojivoto-voting-svc/gen/proto"
+	"github.com/edgelesssys/emojivoto/emojivoto-voting-svc/voting"
+	"google.golang.org/grpc"
+)
+
+type PollServiceServer struct {
+	poll voting.Poll
+}
+
+func (pS *PollServiceServer) vote(shortcode string) (*pb.VoteResponse, error) {
+	err := pS.poll.Vote(shortcode)
+	return &pb.VoteResponse{}, err
+}
+
+func (pS *PollServiceServer) VoteDoughnut(_ context.Context, _ *pb.VoteRequest) (*pb.VoteResponse, error) {
+	//return nil, fmt.Errorf("ERROR")
+	return pS.vote(":doughnut:")
```

Comment:
Isn't this and the SecurityVersion the only two changes we need?

Reply:
We also need to update the import paths in the server.go to avoid building the same binary as before

Reply:
My idea was that the patch is actually a patch, so it should build the same binary as before.
I'd add a patch and a Makefile target that applies the patch and builds the service as before.

Reply:
I see, i will look into it and change the Makefile / Dockerfile accordingly